### PR TITLE
Pull request for Issue2126:  Add button to scanDataView that opens exportData folder

### PR DESCRIPTION
**Pull request by epengr**
Added a button in scanDataView that opens a new system file viewer window at user's exportData folder.

### CODE REVIEW

**Comment by epengr** [comment]
And while you're at it, @iainworkman, could you glance at this little one too?

**Comment by iainworkman** [comment]
Looks good to me.
Time Estimate: 10 minutes

File: source/ui/dataman/AMScanDataView.h [hunk 1]
```diff
@@ -67,6 +69,8 @@ protected slots:
 	void onExportScans();
 	/// Handles the user choosing to show selected scan configurations
 	void onShowScanConfiguration();
+	/// Handles the user choosing to open user export data folder
+	void onexportDataFolderButtonClicked();
```
Comment (dretrex) [inline source/ui/dataman/AMScanDataView.h]: Camel case typo.

**Comment by dretrex** [comment]
Looks good other than one trivial change.

**Comment by epengr** [comment]
Done.  Ready to go.

Time Estimate: 2 min

**Pull request merged**